pkg/rtc/mediatrack.go

```diff
@@ -82,12 +85,32 @@ type MediaTrackParams struct {
 	Logger              logger.Logger
 }
 
+type ConnectionStat struct {
+	PacketsLost  uint32
+	Delay        uint32
+	Jitter       uint32
+	TotalPackets uint32
+}
+
+type ConnectionStats struct {
+	Curr        *ConnectionStat
+	Prev        *ConnectionStat
+	LastUpdated time.Time
+	Score       float64
+}
+
+func NewConnectionStats() *ConnectionStats {
+	return &ConnectionStats{Curr: &ConnectionStat{}, Prev: &ConnectionStat{}}
+}
+
 func NewMediaTrack(track *webrtc.TrackRemote, params MediaTrackParams) *MediaTrack {
 	t := &MediaTrack{
-		params:   params,
-		ssrc:     track.SSRC(),
-		streamID: track.StreamID(),
-		codec:    track.Codec(),
+		params:            params,
+		ssrc:              track.SSRC(),
+		streamID:          track.StreamID(),
+		codec:             track.Codec(),
+		connectionStatsUp: NewConnectionStats(),
+		done:              make(chan bool),
```

Comment:
nit: empty struct instead of bool since we don't use the value.

```suggestion
		done:              make(chan struct{}),
```